dist/docker/debian/build_docker.sh

```diff
@@ -22,6 +22,13 @@ if uname -m | grep aarch64 ; then
   arch="arm64"
 fi
 
+if [ "$product" = "scylla" ]; then
+  summary_label="ScyllaDB Open Source"
+  description_label=""
```

Comment:
Keep just this part and move the enterprise part to the enterprise repo.

Add a description here.

Reply:
Do you mean get rid of the `if` and open another PR under enterprise repo?